NetworkClientUtils.java

```diff
@@ -0,0 +1,196 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
```

Comment:
where is this file!? In the root directory? Why do I see two `NetworkClientUtils.java`?